scripts/android/publish-module.gradle

```diff
@@ -33,7 +34,7 @@ afterEvaluate {
 
                 // Two artifacts, the `aar` (or `jar`) and the sources
                 if (project.plugins.findPlugin("com.android.library")) {
-                    from components.release
+                    artifact("$buildDir/outputs/aar/${project.getName()}-release.aar")
```

Comment:
@Steven0351 Indeed :)

Reply:
I totally missed this and I was even actively looking for it 😆

Reply:
I blame the way Github ordered all the changes, although its just A-Z dir name. But still